components/bl-treeSelect-component/src/index.js

```diff
@@ -54,14 +50,14 @@ export default function TreeSelectComponent({ component, eventHandlers }) {
 
   return (
     <TreeSelect
-      ref={ treeSelectRef }
+      ref={ el => elRef.current = el?.getElement() }
```

Comment:
Do we really need to use assignment here?

As far as I know, `foo = bar` returns `undefined`
So the `ref` prop is also will be undefined?

Reply:
if we pass elRef like this

Reply:
move it to an fn with using useCallback